### ISSUE TITLE
Responsive Items

### ISSUE THREAD

**Issue by javivelasco** [react-toolbox/react-toolbox#224]
I'm opening this issue to track the progress about responsive design. Adding mixins like suggested [here](https://github.com/react-toolbox/react-toolbox/issues/197) is a good starting point. I'll be editing this issue to have a list of next actions. Help is very welcome.

**Comment by nathanmarks** [comment]
AWESOME :) 

@javivelasco When you next get a chance i'd like to ask you a couple of Qs about your plans moving forward

**Comment by javivelasco** [comment]
Great, reach me though Discord!

**Comment by pixiez** [comment]
@javivelasco @soyjavi exciting to know you are making all components responsive in the future release, may I ask if this is still on track? roughly when you think this will be available? cheers

**Comment by javivelasco** [comment]
Yes it is @pixiez, but personally I'm not having too much time lately. Hopefully this week I'll try to open a new branch to work on it. It shouldn't take too long

**Comment by pixiez** [comment]
@javivelasco thats would be very exciting features to have! thanks!

**Comment by zuhair-naqvi** [comment]
Any progress?

**Comment by javivelasco** [comment]
Not too much... Sadly I'm not having too much time to dedicate to the project lately, sorry :(

**Comment by renato** [comment]
I'm sorry to ask again, but... is there any progress?

**Comment by javivelasco** [comment]
Nope, sorry. During time I can spend on React Toolbox these days I'm mostly fixing bugs and small feature requests. I need help to deal with this

**Comment by cam-stitt** [comment]
Great work on the package so far, it's the best material implementation I've seen so far. 

I've spent a little time looking at this. Any more thoughts on how his could be done? I noticed you have the break points already setup. Were you thinking global or module media queries?

**Comment by javivelasco** [comment]
Thanks for the time you're spending on it! :) I did not think too much about it but the breakpoints are already specified because they are documented in Material Design Spec. The best would be to get advantage of those variables and try to make every component look good on each screen size. But again, I did not think too much about it actually.

**Comment by vmsoftware** [comment]
Hi everyone,

I think this would be great for implement in react toolbox:

[](http://materializecss.com/grid.html)

It's really easy to implement a responsive layout only need add a S, M or L letter and a number between 1 and 12 like the google grid specification, as follow:

```
<div class="s12 m6 l4">
   some content or component
</div>
```

It mean have a div that use all 12 columns at small screen(100%), use 6 in medium screem(50%) and use 4 on large screem(33.3%). As you can see it's really easy and usefull.

Best regards!

**Comment by shivasurya** [comment]
any progress on responsive break points ?

**Comment by javivelasco** [comment]
We already have break points but there are no definitions to make all components look good into smaller screens. Since I'm not using the project in a responsive environment, contributions about this are very welcome!

**Comment by sai-prasanna** [comment]
@javivelasco  Need to do touch handling for NavDrawers, for closing them in mobile environments.

**Comment by javivelasco** [comment]
@sai-prasanna this should be easy. Check the CSS next implementation that has a complete rewrite of the `Layout` and check out to add a handler different from Click

**Comment by Prior99** [comment]
What is the current status of this? Can responsiveness be expected in the near future or should we use different implementation of material design for React instead?
